src/SFA.DAS.Assessors.Api/Models/GetCourseListItem.cs

```diff
@@ -5,8 +5,12 @@ namespace SFA.DAS.Assessors.Api.Models
 {
     public class GetCourseListItem
     {
-        public int Id { get; set; }
+        public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
+        public int LarsCode { get; set; }
         public string Title { get; set; }
+        public decimal? Version { get; set; }
```

Comment:
you've got this as a string in courses I think? to reflect 1.10 etc.

Reply:
There's an open PR to change version from decimal to string but that will be part of the next release.